specification/agfood/data-plane/Microsoft.AgFoodPlatform/preview/2021-03-31-preview/examples/Boundaries_SearchByFarmerId.json

```diff
@@ -0,0 +1,87 @@
+{
+  "parameters": {
+    "farmerId": "FARMER123",
+    "api-version": "2021-03-31-preview",
+    "body": {
+      "ids": [
+        "BOUNDARY123"
+      ],
+      "names": [
+        "John Smith's Boundary"
+      ],
+      "propertyFilters": [
+        "RetailerId"
+      ],
+      "statuses": [
+        "Active"
+      ],
+      "minCreatedDateTime": "2021-04-22T12:53:39Z",
+      "maxCreatedDateTime": "2021-04-22T12:53:39Z",
+      "minLastModifiedDateTime": "2021-04-22T12:53:39Z",
+      "maxLastModifiedDateTime": "2021-04-22T12:53:39Z",
+      "$maxPageSize": 50,
+      "$skipToken": "string",
+      "isPrimary": true,
+      "parentType": "field",
+      "parentIds": [
+        "FIELD123"
+      ],
+      "minAcreage": 200,
+      "maxAcreage": 800,
+      "intersectsWithGeometry": {
```

Comment:
We need to add the complete geometry i.e. the coordinates

Reply:
Fixed